plugins/logitech-tap/fu-logitech-tap-sensor-device.c

```diff
@@ -312,6 +312,7 @@ fu_logitech_tap_sensor_device_init(FuLogitechTapSensorDevice *self)
 	fu_device_set_version_format(FU_DEVICE(self), FWUPD_VERSION_FORMAT_TRIPLET);
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_UPDATABLE);
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_SIGNED_PAYLOAD);
+	fu_device_add_vendor_id(FU_DEVICE(self), "USB:0x0872");
```

Comment:
I think this is not needed when https://github.com/fwupd/fwupd/pull/7762 is merged.